docs/guides/Redux-Integration.md

```diff
@@ -1,6 +1,6 @@
 # Redux Integration
 
-To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
+To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Once you pass your own navigation prop to the navigator, the navigator loses defualt [`navigation`](https://reactnavigation.org/docs/navigators/navigation-prop) prop. So we have to pass the two basic and must have properties of `navigation` prop namely `state` and `dispatch`, else we will not be able to access `this.props.navigation.state` and `this.props.navigation.dispatch` in the components. The state will be fed from the reducer assigned to handle navigation state and the `dispatch` will be redux's default `dispatch`. Thus we will be able to dispatch normal redux actions using `this.props.navigation.dispatch(ACTION)`, reducer will update the navigation state on the basis of dispatched action, the new navigation state will then be passed to the navigator. Following is the detailed explanation of how things work:
```

Comment:
A couple things:

Reply:
Thanks for the review. 

Regarding point-2, I have added this line because this issue #1405 states that we need to explicitly tell the people that the default navigation prop is replaced by our own navigation prop. I used the word **loses** so that the people will get to know that its there duty to pass the navigation properties like `state`, `dispatch`. As we are passing `{state, dispatch}` and not `navigation` in the docs, so it's confusing to the people.

Do let me know what you think, I will then push a new commit.

Reply:
Gotcha; how about then "By passing your own navigation prop to the navigator, it will replace the default [`navigation`](https://reactnavigation.org/docs/navigators/navigation-prop) prop." ?

Reply:
That's fine too, but what I think is we need to explicitly tell the people that the default `navigation` prop is destroyed so that they know that they need to pass their own properties.  Suppose if they only pass `state`, `this.props.navigation.dispatch` is still not accessible , they might think where is `dispatch` gone, they might get the impression that the default `dispatch` should be there because they have only tinkered with the `state` property of `navigation` prop? 

PS: Though I don't think not sending `dispatch` is a use case. 
What say?

Reply:
Ah gotcha. Yeah then I guess maybe even add this comment in that phrase to make it even more clear.